CS3219-AY2223S1/cs3219-project-ay2223s1-g12
# Implement UI and update backend for cookies for logout

Fixes #66 
Depends on #63 
**Summary**
* Current logout operations in the backend do not interact with our token cookies that we set in the user's browser when they log in, and thus updates are needed to do so.
* Implement removal of our token cookies from the user's browser upon logout.
* Update logout function in backend to read target token from cookies instead of request body.
* Add skeleton UI destination page after logout is successful.
* Add home page UI as placeholder for matching-service.
* Implement logout button and logic to home page

Steps to test:
1. Navigate to `http://localhost:3000/login` and proceed to sign in.
2. Upon redirection to home page, open browser's developer options > Application tab. Observe that two cookies 'token' and `refreshToken` are saved on the browser.
3. Process to click on logout button on the home page.
4. Upon redirection to logged out page, Observe in the developer options > Application tab that the above two cookies are indeed deleted.
